src/ejabberd_admin.erl

```diff
@@ -442,22 +442,33 @@ get_loglevel() ->
 %%% Purge DB
 %%%
 
--spec delete_expired_messages(jid:lserver()) -> {ok, iolist()} | {error, iolist()}.
+-spec delete_expired_messages(mongooseim:host_type()) -> {ok, iolist()} | {error, iolist()}.
```

Comment:
This is still taking a `lserver`, so the type shouldn't have changed.

Reply:
Good catch! I changed it back and forth and forgot to revert this one.